starcheck/calc_ccd_temps.py

```diff
@@ -110,7 +111,7 @@ def get_ccd_temps(oflsdir, outdir='out',
     :param outdir: output directory for plots
     :param json_obsids: file-like object or string containing JSON of
                         starcheck Obsid objects (default='<oflsdir>/starcheck/obsids.json')
-    :param model_spec: xija ACA model spec file (default=package aca_spec.json)
+    :param model_spec: xija ACA model spec file (default=chandra_models current aca_spec.json)
```

Comment:
It's a little odd that the `model_spec` kwargs exists in this function but there is no way to set it from the `starcheck` application. Is there some use case for running `starcheck.calc_ccd_temps` outside of starcheck run?

BTW, in the docstring above there is a reference to `sybase`, which I assume is outdated.

Reply:
I think the only use case for outside of starcheck run is for testing, like you repaired and then did in https://github.com/sot/starcheck/pull/344 .  But I figured that there was no real driver to *cut* the parameters but we could simplify if you'd like.